File: java/MASTER_SLAVE_WHOLE_PART_PROXY/Reservacion.java
public class Reservacion {
    //Atributos
    private DatosCliente datosCliente;
    private DatosCompra datosCompra;
    private DatosFuncion datosFuncion;

    //Constructor
    public Reservacion(){}

    //Métodos
    public String mostrarReservacion(){
        return datosCliente.getDatosCliente() + datosCompra.getDatosCompra() + datosFuncion.getDatosFuncion();
    }

    public void setDatosCliente(DatosCliente dc){
        this.datosCliente = dc;
    }

    public void setDatosCompra(DatosCompra dcc){
        this.datosCompra = dcc;
    }

    public void setDatosFuncion(DatosFuncion df){
        this.datosFuncion = df;
    }
}

File: php/MASTER_SLAVE_WHOLE_PART_PROXY/principal/Reservacion.php
<?php

class Reservacion{
    
    private $datosCliente;
    private $datosCompra;
    private $datosFuncion;
    

    public function __construct(){
        
    }

    public function setDatosCliente($dc){
        $this->datosCliente = $dc;
    }
    public function setDatosCompra($dc){
        $this->datosCompra = $dc;
    } 
    public function setDatosFuncion($df){
        $this->datosFuncion = $df;
    }

    public function mostrarReservacion(){
        return $this->datosCliente->getDatosCliente().$this->datosFuncion->getDatosFuncion().$this->datosCompra->getDatosCompra();
    }


}

